internal/statusreaders/job_status_reader.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright The Helm Authors.
```

Comment:
Apache license requires us to retain the Flux copyright on these files.

The fluxcd/kustomize-controller project does not have a NOTICE file, so it's sufficient to add the Helm copyright to the existing Flux license header on this and any other derived files.

```suggestion
Copyright The Helm Authors.
This file was initially copied and modified from
    https://github.com/fluxcd/kustomize-controller/blob/main/internal/statusreaders/job.go
Copyright 2022 The Flux authors
```